storage/google/cloud/storage/client.py

```diff
@@ -211,10 +211,16 @@ def create_bucket(self, bucket_name):
         :type bucket_name: str
         :param bucket_name: The bucket name to create.
 
+        :type requester_pays: bool
+        :param requester_pays: (Optional) Whether requester pays for
+            API requests for this bucket and its blobs.
```

Comment:
We've typically tried to either indent 4 spaces for **all content** or to line up the continuation lines with the previous content.